app/src/main/res/layout/activity_login.xml

```diff
@@ -2,92 +2,107 @@
     android:layout_height="match_parent"
     android:layout_width="match_parent"
     android:orientation="vertical">
 
+    <include
+        layout="@layout/toolbar"
+        android:visibility="gone" />
+
     <androidx.core.widget.NestedScrollView
-        android:fitsSystemWindows="true"
+        android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_width="match_parent">
+        android:orientation="vertical">
 
         <LinearLayout
             android:id="@+id/ll_login"
-            android:layout_height="wrap_content"
             android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_marginTop="@dimen/layout_padding_150dp"
             android:orientation="vertical"
-            android:paddingEnd="@dimen/login_padding_right"
-            android:paddingLeft="@dimen/login_padding_left"
-            android:paddingRight="@dimen/login_padding_right"
-            android:paddingStart="@dimen/login_padding_left"
-            android:paddingTop="@dimen/login_padding_top">
-
-            <ImageView
-                android:layout_gravity="center_horizontal"
-                android:layout_height="@dimen/login_height"
-                android:layout_marginBottom="@dimen/login_margin_bottom"
-                android:layout_width="wrap_content"
-                android:src="@drawable/mifos_logo"/>
-
-            <!-- Username  Label -->
-            <com.google.android.material.textfield.TextInputLayout
-                android:id="@+id/til_username"
+            android:padding="@dimen/layout_padding_30dp">
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:fontFamily="@font/productsans"
+                android:text="@string/app_name"
+                android:textAlignment="center"
+                android:textColor="@color/white"
+                android:textSize="@dimen/text_size_35sp"
+                android:gravity="center_horizontal" />
+
+            <LinearLayout
+                android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_marginBottom="@dimen/default_margin"
-                android:layout_marginTop="@dimen/default_margin"
-                android:layout_width="match_parent">
-
-                <EditText
-                    style="@style/Base.TextAppearance.AppCompat.Medium"
-                    android:drawableLeft="@drawable/ic_wrapped_person_black_24dp"
-                    android:drawablePadding="@dimen/layout_padding_8dp"
-                    android:drawableStart="@drawable/ic_wrapped_person_black_24dp"
-                    android:hint="@string/username"
-                    android:inputType="textEmailAddress"
+                android:layout_marginTop="@dimen/layout_padding_65dp"
+                android:orientation="vertical">
+
+
```

Comment:
remove the extra line. Format your code before pushing in JetBrains IDE.

Reply:
okay.